web/src/components/storage/SearchedDeviceMenu.tsx

```diff
@@ -172,10 +170,13 @@ const ChangeDeviceMenuItem = ({
   device,
   ...props
 }: ChangeDeviceMenuItemProps): React.ReactNode => {
+  const onlyOneOption = UseOnlyOneOption(modelDevice);
```

Comment:
NP: not part of your changes and probably simply a typing mistake, but we usually use PascalCase for components, not for hooks. I.e., should be `useOnlyOneOption` or not use `use` prefix if it is not a hook.